DepositeCalcTests/Tests/SettingsPageTests.cs

```diff
@@ -86,11 +69,11 @@ public void DefaultCurrencyTest(string expectedCurrency)
         {
             // Arrange
             var calculatorPage = new CalculatorPage(driver);
-            var settingsPage = new SettingsPage(driver);
 
             // Act
             settingsPage.Set(expectedCurrency);
             string expectedCarrencyEmblem = expectedCurrency.Substring(0, 1);
+            new WebDriverWait(driver, TimeSpan.FromSeconds(5)).Until(_ => calculatorPage.IsOpened());
```

Comment:
1. Please move 'wait' to the calculator page. 
2. When you save settings, the calculator page is opened. So you can return this page at the end of the 'Set' method.
3. 'Arrange' is a good place to initialize test data/variables.
```suggestion
            // Arrange
            string expectedCarrencyEmblem = expectedCurrency.Substring(0, 1);
            
            // Act
            CalculatorPage calculatorPage = settingsPage.Set(expectedCurrency);
            calculatorPage.WaitForReady();
```